scripts/test.ts

```diff
@@ -1,26 +1,23 @@
 const argv = require("minimist")(process.argv.slice(2))
 
-import * as fs from "fs"
-import * as path from "path"
 import {execSync} from "child_process"
 import chalk from "chalk";
-import ora from "ora";
 
-const EXAMPLES_DIR = path.resolve("examples")
+// const EXAMPLES_DIR = path.resolve("examples")
 
-function listDirs(dir: string) {
-  return fs
-    .readdirSync(dir)
-    .filter(dirname => fs.lstatSync(path.join(dir, dirname)).isDirectory())
-}
-
-function installExample(example) {
-  const spinner = ora("Installing " + example)
-  execSync("yarn", {
-    cwd: path.join(EXAMPLES_DIR, example)
-  })
-  spinner.succeed()
-}
+// function listDirs(dir: string) {
+//   return fs
+//     .readdirSync(dir)
+//     .filter(dirname => fs.lstatSync(path.join(dir, dirname)).isDirectory())
+// }
+//
+// function installExample(example) {
+//   const spinner = ora("Installing " + example)
+//   execSync("yarn", {
+//     cwd: path.join(EXAMPLES_DIR, example)
+//   })
+//   spinner.succeed()
+// }
```

Comment:
hoping to make it work, one day

Reply:
Most likely, such a commented code will not be fixed ever (below we have such an example). So it would be better to remove it and create an issue describing what should be done

Reply:
💪